# Goal
sensor_msgs/Image image
---
# Result
uint32 view_cell
---
# Feedback
#float32 time_exec
uint32 time_exec